src/components/code_editor/code_editor.tsx

```diff
@@ -36,6 +36,20 @@ export interface EuiCodeEditorProps
   setOptions: IAceEditorProps['setOptions'];
   cursorStart?: number;
   'data-test-subj'?: string;
+  /**
+   * Select theme for code-editor
+   */
+  theme?:
+    | 'monokai'
+    | 'tomorrow'
+    | 'github'
+    | 'kuroir'
+    | 'twilight'
+    | 'xcode'
+    | 'solarized_light'
+    | 'solarized_dark'
+    | 'terminal'
+    | 'textmate';
```

Comment:
```suggestion
  /**
   * Select the `brace` theme
   * The matching theme file must also be imported from `brace` (e.g., `import 'brace/theme/github';`)
   */
  theme?: IAceEditorProps['theme'];
```
We can use the `react-ace` interface for `theme` directly here to avoid maintaining the list of available themes (which comes from `brace`).
https://github.com/securingsincity/react-ace/blob/v7.0.5/src/ace.tsx#L25-L26